api/src/main/java/de/oliver/fancyholograms/api/utils/FloodgateChecker.java

```diff
@@ -0,0 +1,19 @@
+package de.oliver.fancyholograms.api.utils;
+
+import org.bukkit.entity.Player;
+import org.geysermc.floodgate.api.FloodgateApi;
+
+public class FloodgateChecker {
+
+    public static boolean isBedrockPlayer(Player player) {
+        if (player == null) {
+            return false;
+        }
+
+        try {
+            return FloodgateApi.getInstance().isFloodgateId(player.getUniqueId());
+        } catch (Throwable ignored) {
+            return false;
+        }
+    }
```

Comment:
As we only need this for online players it would be better to use `isFloodgatePlayer` there is potential that this current code wouldn't work with linked accounts.
```suggestion
    public static boolean isBedrockPlayer(@NotNull Player player) {
        return FloodgateApi.getInstance().isFloodgatePlayer(player.getUniqueId());
    }
```

Reply:
> As we only need this for online players it would be better to use `isFloodgatePlayer` there is potential that this current code wouldn't work with linked accounts.
> 
> ```suggestion
> 
>     public static boolean isBedrockPlayer(@NotNull Player player) {
> 
>         return FloodgateApi.getInstance().isFloodgatePlayer(player.getUniqueId());
> 
>     }
> 
> ```

for some reason on testing, with a velocity server and geyser standalone with floodgate on the backend server. this was returning false when indeed the player was joined through geyser on standalone and their uuid became floodgate uuid. not sure why it was returning false.